### ISSUE TITLE
CPU timer: no timer used.

### ISSUE THREAD

**Issue by DragonPara** [DrTimothyAldenDavis/SuiteSparse#822]
+ cmake command
`cmake -G"Ninja Multi-Config" -DCMAKE_CXX_COMPILER=cl -DCMAKE_C_COMPILER=cl -DCMAKE_Fortran_COMPILER=ifort -DSUITESPARSE_USE_OPENMP=ON -DSUITESPARSE_ENABLE_PROJECTS="umfpack" -DCMAKE_INSTALL_PREFIX=F:\umfpackWork\suitesparse7.7 --fresh ..`
+ compile command
cmake --build . --config Release

+ test code
```c++
#include <stdio.h>
#include "umfpack.h"
int n = 5;
int Ap[] = {0, 2, 5, 9, 10, 12};
int Ai[] = {0, 1, 0, 2, 4, 1, 2, 3, 4, 2, 1, 4};
double Ax[] = {2., 3., 3., -1., 4., 4., -3., 1., 2., 2., 6., 1.};
double b[] = {8., 45., -3., 3., 19.};
double x[5];
int main(void)
{
    double *null = (double *)NULL;
    int i;
    void *Symbolic, *Numeric;
    double infos[UMFPACK_INFO];
    (void)umfpack_di_symbolic(n, n, Ap, Ai, Ax, &Symbolic, null, infos);
    (void)umfpack_di_numeric(Ap, Ai, Ax, Symbolic, &Numeric, null, infos);
    umfpack_di_free_symbolic(&Symbolic);
    (void)umfpack_di_solve(UMFPACK_A, Ap, Ai, Ax, x, b, Numeric, null, infos);
    umfpack_di_free_numeric(&Numeric);
    for (i = 0; i < n; i++)
        printf("x [%d] = %g\n", i, x[i]);
    double control[UMFPACK_CONTROL];
    umfpack_di_defaults(control);
    control[UMFPACK_PRL] = 7;
    umfpack_di_report_info(control,infos);
}
```
output
```shell
x [0] = 1
x [1] = 2
x [2] = 3
x [3] = 4
x [4] = 5
UMFPACK V6.3.3 (Mar 22, 2024), Info:
    matrix entry defined as:          double
    Int (generic integer) defined as: int32_t
    BLAS library used: Intel MKL 64lp BLAS (32-bit integers).  size of BLAS integer: 4
    MATLAB: no.
    CPU timer: no timer used.
    number of rows in matrix A:       5
    number of columns in matrix A:    5
    entries in matrix A:              12
...
```

But if I set `-DCMAKE_C_FLAGS=-openmp`, timer will be set to omp_get_wtime()
```shell
CPU timer: omp_get_wtime ( )
```
Why `-DSUITESPARSE_USE_OPENMP=ON` not work?

**Comment by mmuetzel** [comment]
Thank you for reporting the issue.

Afaict, the OpenMP timer function is correctly used when SuiteSparse is configured with `-DSUITESPARSE_USE_OPENMP=ON`. Only the output in the report by UMFPACK is wrong.

PR #823 should be fixing that.

**Comment by DrTimothyAldenDavis** [comment]
Fixed with #823.
